app/models/friendship.rb

```diff
@@ -0,0 +1,4 @@
+class Friendship < ApplicationRecord
+  belongs_to :follower, class_name: 'User'
```

Comment:
(THIS WAS A CODE BRAINSTORM created together with Priya)
- User visits /friends
- User adds email into form
- User clicks button "Add Friend"
- This takes you to a controller action
user_1.followers << user_2 (within button so that user_1 can add a friend (user_2)
FOLLOWEE. || FOLLOWER ||   PRIMARYKEY
===============================
user_1.id  || user_2.id || friendship.id